job/common/src/main/java/alluxio/job/plan/meta/MasterWorkerInfo.java

```diff
@@ -9,7 +9,7 @@
  * See the NOTICE file distributed with this work for information regarding copyright ownership.
  */
 
-package alluxio.job.meta;
+package alluxio.job.plan.meta;
```

Comment:
Why is this in the `plan` package?

Reply:
Because it is used by the PlanCoordinator.

Reply:
It seems to be only be used by `JobMaster`?

Reply:
Hmm.. Ideally, I would want to make JobMaster more Worker agnostic but that's fair. Let me move it out.